server/src/organizations/entities/organization.entity.ts

```diff
@@ -0,0 +1,35 @@
+import { Entity, Column, PrimaryGeneratedColumn } from 'typeorm';
+
+// Considering validators for uniqueness: org_name? ein? both? 
```

Comment:
this is a good question for product
ping the general or design/ux channel and maybe cc: joel so he can point you to someone who would know

Reply:
Heard back from Joel on this. He said to go with the EIN.